master/internal/task/allocation_test.go

```diff
@@ -102,8 +103,17 @@ func TestAllocation(t *testing.T) {
 				}
 				db.On("UpdateAllocationState", mock.Anything).Return(nil)
 				require.NoError(t, system.Ask(self, containerStateChanged).Error())
+
+				beforePulling := time.Now().UTC().Truncate(time.Millisecond)
 				containerStateChanged.Container.State = cproto.Pulling
+				db.On("UpdateAllocationStartTime", mock.Anything).Return(nil)
 				require.NoError(t, system.Ask(self, containerStateChanged).Error())
+				afterPulling := time.Now().UTC().Truncate(time.Millisecond)
+				outOfRange := a.model.StartTime.Before(beforePulling) || a.model.StartTime.After(afterPulling)
+				require.False(t, outOfRange,
+					"Expected start time of open allocation should be in between %q and %q but it is = %q instead",
```

Comment:
if you use a format string you need to use `require.Falsef` instead. and you can drop the `.String()` calls and just use `%s` i think.